static/src/javascripts/bootstraps/enhanced/liveblog.js

```diff
@@ -94,6 +99,30 @@ const initTracking = () => {
 			ophan.record(componentEvent(pinnedBlockId, 'VIEW'));
 		});
 	}
+
+	// pinned post duration tracking
+    let hasBeenSeen = false;
+    const pinnedPostTiming = measureTiming('pinned-post-view-duration');
+    const originalPinnedBlockId = pinnedBlock.dataset.blockId
```

Comment:
What happens if we don't have pinnedBlock here? Should we move this tracking inside the if statement above?

Reply:
Good spot Anna. I put all the logic in a separate function and then am calling the function from within the condition when pinnedPost is available